program1.py

```diff
@@ -0,0 +1,75 @@
+"""
+this is first program to counte the number of vowels, words, sentences, and punctuation marks.
+"""
+import string
+def print_details(vowels,punctuations):
+    """
+    this function is used to print vowels and punctuations in a sentence
+    Parameters:
+           vowels (int) : recieves value of vowels as parameter from main
+           punctuations (int) : recieves count of punctuation marks as parameter from main
+    returns :
+           nothing
+    """
+    print("number of vowels = {}".format(vowels))
+    print("number of punctuation marks  = {}".format(punctuations))
+def count_vowels(test_sentence):
+    """
+    this function is used to count vowels
+    Parameters:
+            test_sentence (str): stores sentence set by user or data read from a file
+    Returns:
+            number_of_vowels (int) : returns count of vowels in sentence
+    """
+    number_of_vowels = 0
+    vowels = ["a","e","i","o","u"]
```

Comment:
Good work!
This can be further improved by converting a list into a set.
The search complexity of the set is O(1) while search complexity of list is O(n)